mkdocs_windmill/base.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
 <head>
   {% set is_top = (page and page.is_homepage) and not force_plain_page %}
-  {% set base_url = base_url or "." %}
+  {% if base_url == "/" %}{% set base_url = "." %}{% endif %}{% set base_url = base_url or "." %}
```

Comment:
I think this could be simpler, as a single set, using `if-else` as here: https://jinja.palletsprojects.com/en/3.0.x/templates/#if-expression